### ISSUE TITLE
Is there a way to update variable without using setState, notifyListeners or any external packages?

### ISSUE THREAD

**Issue by rafaelovandov** [buttonsrtoys/mvvm_plus#33]

**Comment by buttonsrtoys** [comment]
Yes. Variables in ViewModels are often called "properties" when using MVVM. To declare a variable (property), you can

      late final count = createProperty<int>(0);
 
And then update the property like

    count = 5;

or 

    count++;

`mvvm_plus` uses notifyListeners under the hood to notify listeners that the value has changed.

**Comment by JCKodel** [comment]
Just a word of advice: although `createProperty<T>()` is easy to use, it publishes it mutability (i.e.: I can change the value of a property outside the class that declares it, which is very very very bad).

A better approach is always use a private changer:

```dart
int _count = 0;
int get count => _count;

void someMethodThatMutatesCount() {
  ...
  if(_count != newCountValue) {
    _count = newCountValue;
    buildView();
  }
  ...
}
```

First, it won't rebuild if nothing has changed.

Second, you can't change `count` outside of that view model.

More code? Yes. But A LOT safer.
